src/parser/whamm.pest

```diff
@@ -52,14 +52,15 @@ statement = { ( ( fn_call | assignment ) ~ ";" )+ }
 // ---- Expressions ----
 // =====================
 
-expr = { operand ~ (BINOP ~ operand)* }
+expr = { prefix? ~ operand ~ (BINOP ~ prefix? ~ operand)* }
 
+neg = { "!" }
```

Comment:
I would add a:
```
UNOP = _{ neg }
    neg = { "!" }
```

Like it does below (increases readability when adding more unops)